dockerfiles/Dockerfile.goreleaser

```diff
@@ -0,0 +1,8 @@
+FROM scratch
+
```

Comment:
The `Dockerfile.guac-cont` has `ca-certificates` packages. maybe this one should too?

Reply:
good point...ca-certs added!